atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitions.java

```diff
@@ -150,7 +153,6 @@ static CfDef getStandardCfDef(String keyspace, String internalTableName) {
         cf.setKey_validation_class("org.apache.cassandra.db.marshal.BytesType");
         cf.setCompaction_strategy_options(new HashMap<String, String>());
```

Comment:
this line was the one obliterating what we already set above

Reply:
should be add the "sstable_size_in_mb" property to the map here? How will it be set now?